src/wp-includes/class-wp-object-cache.php

```diff
@@ -142,6 +142,34 @@ protected function _exists( $key, $group ) {
 		return isset( $this->cache[ $group ] ) && ( isset( $this->cache[ $group ][ $key ] ) || array_key_exists( $key, $this->cache[ $group ] ) );
 	}
 
+	/**
+	 * Serves as a utility function to determine whether a key is valid.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param int|string $key   Cache key to check for validity.
+	 * @return bool Whether the key is valid.
+	 */
+	protected function _valid_key( $key ) {
+		if ( is_int( $key ) ) {
+			return true;
+		}
+
+		if ( is_string( $key ) && trim( $key ) !== '') {
+			return true;
+		}
+
+		$type = str_replace( 'string', 'empty string', gettype( $key ) );
+
+		_doing_it_wrong(
+			debug_backtrace()[1]['function'],
```

Comment:
Better use `debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 2 )[1]['function'];` for (siginificantly) better performance

Reply:
Thanks, addressed in d896c043d8054bca7826148301608124a8895f6a.